index.js

```diff
@@ -15,9 +15,75 @@ mongoose
     // Before adding any recipes to the database, let's remove all existing ones
     return Recipe.deleteMany()
   })
+  .then(() => Recipe.syncIndexes())
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+
+    Recipe
+
+      //ITERATION 2
+      //add a new recipe document to the database by calling the Model.create static, passing it the recipe details as an object.
+
+      .create([
+        {
+          title: "Boniato",
+          level: "Easy Peasy",
+          ingredients: ["Boniato", "Aceite", "Especias"],
+          cuisine: "Mediterranea",
+          dishType: "other",
+          image: "https://imag.bonviveur.com/foto-de-la-portada-de-los-bastones-de-boniato-al-horno.jpg",
+          duration: 45,
+          creator: "Miguel Ortega",
+          created: new Date(1995, 11, 17, 3, 24, 0)
+        },
+      ])
+
+      //ITERATION 3
+      /*
+      importing an array of recipes form the data.json file. Using the Model.
+      insertMany static, you should add the entire array to the database.
+      After inserting the documents, print the title of each recipe to the console.
+      */
+
+      .then(() => {
+        return Recipe.insertMany(data)
+      })
```

Comment:
- Cuidado aquí porque estás anidando promesas una dentro de otra (el `then` de `insertMany` está dentro del `then` de `create`. La forma de solucionar esto es haciendo que el primer `then` haga `return` de la promesa `Recipe.create()` y gestionar la respuesta con el `then` desde fuera.

```suggestion
.then(() => {
    return Recipe
      .create(
        {
          title: "Boniato",
          level: "Easy Peasy",
          ingredients: ["Boniato", "Aceite", "Especias"],
          cuisine: "Mediterranea",
          dishType: "other",
          image: "https://imag.bonviveur.com/foto-de-la-portada-de-los-bastones-de-boniato-al-horno.jpg",
          duration: 45,
          creator: "Miguel Ortega",
          created: new Date(1995, 11, 17, 3, 24, 0)
        },
      )
  })
  .then((res) => {
    console.log(res)
    return Recipe.insertMany(data)
  })
```